pkg/csi/service/common/common_controller_helper.go

```diff
@@ -40,17 +39,15 @@ func ValidateCreateVolumeRequest(ctx context.Context, req *csi.CreateVolumeReque
 	// Volume Name
 	volName := req.GetName()
 	if len(volName) == 0 {
-		msg := "Volume name is a required parameter."
-		log.Error(msg)
-		return status.Error(codes.InvalidArgument, msg)
+		return logger.LogNewErrorCode(log, codes.InvalidArgument, "Volume name is a required parameter")
```

Comment:
In golang, returned error string should start with lower case.  Can you change this to "volume ..."?